include/device.h

```diff
@@ -667,9 +652,7 @@ static inline bool device_is_ready(const struct device *dev)
  */
 #define Z_DEVICE_STATE_DEFINE(node_id, dev_name, pm_action_cb)		\
```

Comment:
pm_action_cb is no longer used by `Z_DEVICE_STATE_DEFINE()`.  pm_action_cb should also be removed from `Z_DEVICE_DEFINE_PRE()`

Reply:
fixed